tests/bin/distrib-name/run.t

```diff
@@ -4,82 +4,82 @@ dune-project. This goes with the fix for #320, where:
 - dune distrib ignores this name and fails, complaining that you
 should set a name.
 
-    $ mkdir liba libb
-    $ cat > CHANGES.md << EOF \
-    > ## 0.42.0\
-    > \
-    > - Some other feature\
-    > \
-    > EOF
-    $ echo "(library (public_name liba))" > liba/dune
-    $ echo "(library (public_name libb))" > libb/dune
-    $ cat > liba.opam << EOF \
-    > opam-version: "2.0" \
-    > EOF
-    $ cp liba.opam libb.opam
-    $ touch README LICENSE
-    $ echo "(lang dune 2.7)" > dune-project
-    $ cat > .gitignore << EOF \
-    > _build\
-    > .formatted\
-    > .mdx\
-    > /dune\
-    > run.t\
-    > EOF
-    $ git init 2> /dev/null . > /dev/null
-    $ git config user.name "dune-release-test"
-    $ git config user.email "pseudo@pseudo.invalid"
-    $ git add liba/* libb*/ CHANGES.md README LICENSE *.opam dune-project .gitignore
-    $ git commit -m 'Commit.' > /dev/null
+  $ mkdir liba libb
+  $ cat > CHANGES.md << EOF
+  > ## 0.42.0
+  > 
+  > - Some other feature
+  > 
+  > EOF
+  $ echo "(library (public_name liba))" > liba/dune
+  $ echo "(library (public_name libb))" > libb/dune
+  $ cat > liba.opam << EOF
+  > opam-version: "2.0"
+  > EOF
+  $ cp liba.opam libb.opam
+  $ touch README LICENSE
+  $ echo "(lang dune 2.7)" > dune-project
+  $ cat > .gitignore << EOF
+  > _build
+  > .formatted
+  > .bin
+  > /dune
+  > run.t
+  > EOF
+  $ git init 2> /dev/null . > /dev/null
+  $ git config user.name "dune-release-test"
+  $ git config user.email "pseudo@pseudo.invalid"
+  $ git add liba/* libb*/ CHANGES.md README LICENSE *.opam dune-project .gitignore
+  $ git commit -m 'Commit.' > /dev/null
 
 Try dune-release distrib with no project name.
 
-    $ dune-release distrib --skip-lint
-    [-] Building source archive
-    dune-release: [ERROR] cannot determine distribution name automatically: add (name <name>) to dune-project
-    [1]
+  $ dune-release distrib --skip-lint
```

Comment:
Even better, since the error messages are output to `stderr` we can just pipe `stdout` to `/dev/null` and get all the relevant errors that we're looking for.